### DISCUSSION TITLE
How to set Widget Preview

### DISCUSSION THREAD

**Discussion by tech-dhruv** [ABausG/home_widget#279] [Help and Q&A] [answered]
![image](https://github.com/user-attachments/assets/65b222cb-fd20-4f2b-9ada-46ff9775638a)

How to Show Widget Preview?

**Comment by ABausG** [comment, marked as answer]
https://developer.android.com/develop/ui/views/appwidgets/enhance#add-scalable-widget-previews

**Comment by Cyrille37** [comment]
See also #359
